dockstore-webservice/src/main/java/io/dockstore/webservice/core/FileFormat.java

```diff
@@ -42,7 +43,10 @@
 @ApiModel(value = "FileFormat", description = "This describes an input or output file format that is associated with an entry in the dockstore")
 @Entity
 @Table(name = "fileformat")
-@NamedQuery(name = "io.dockstore.webservice.core.FileFormat.findByFileFormatValue", query = "SELECT l FROM FileFormat l WHERE l.value = :fileformatValue")
+@NamedQueries({
```

Comment:
I guess you don't need this change any more (you added some queries, then decided not to?). Fine to keep though